### DISCUSSION TITLE
Comment créer un bot de vérification d'humain pour l'accès au groupe ?

### DISCUSSION THREAD

**Discussion by Marco9579** [python-telegram-bot/python-telegram-bot#4173] [Q&A]
J'ai réussi à créer le bot mais pour le moment il demande une équation simple et attend une réponse précise mais lorsque je l'ajoute dans mon groupe telegramme il pose la question et lorsqu'on répond il ne passe rien je doit utilisé la commande /start pour qu'il réponde je suis un peut bloqué...

import telebot
from telebot import types

telegram_api_key = 'My key API'

bot = telebot.TeleBot(telegram_api_key)

# Handler pour les commandes start et help
@bot.message_handler(commands=['start', 'help'])
def send_welcome(message):
    msg = bot.reply_to(message, "Bonjour! Pour vérifier que vous êtes un humain, veuillez répondre à cette question : 3 + 4 = ?")
    bot.register_next_step_handler(msg, process_verification_step)

def process_verification_step(message):
    try:
        chat_id = message.chat.id
        answer = message.text.strip()
        
        if answer == "7":
            bot.send_message(chat_id, "Vérification réussie. Bienvenue!")
        else:
            msg = bot.reply_to(message, "Vérification échouée. Veuillez essayer de nouveau. 3 + 4 = ?")
            bot.register_next_step_handler(msg, process_verification_step)
    except Exception as e:
        bot.reply_to(message, 'Oups! Quelque chose s\'est mal passé.')

# Lance le bot
bot.infinity_polling()

**Comment by Poolitzer** [comment]
Hi, english only, and wrong library:)

**Reply by Marco9579** [reply]
Sorry, thank !
